packages/span-transformations/src/jaeger.ts

```diff
@@ -0,0 +1,70 @@
+import { SpanAttributes, SpanKind } from '@opentelemetry/api';
+import { timeInputToHrTime } from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
+
+const getJaegerValueForTag = (jaegerTagKey, tags) => tags.filter(({ key }) => jaegerTagKey === key)?.[0]?.value;
+const convertJaegerTagsToAttributes = (tags): SpanAttributes => {
+    const spanAttributes: SpanAttributes = {};
+    tags.forEach(({ key, value }) => {
+        spanAttributes[key] = value;
+    });
+    return spanAttributes;
+};
+
+const getOtelKindFromJaegerKind = (jaegerKind: string) => {
+    switch (jaegerKind) {
+        case 'client':
+            return SpanKind.CLIENT;
+        case 'producer':
+            return SpanKind.PRODUCER;
+        case 'server':
+            return SpanKind.SERVER;
+        case 'consumer':
+            return SpanKind.CONSUMER;
+        default:
+            return SpanKind.INTERNAL;
+    }
+};
+
+const getParentSpanID = (jaegerSpan) => {
+    let parentSpanId = '';
+    try {
+        parentSpanId = jaegerSpan.references?.filter(({ refType }) => refType === 'CHILD_OF')[0].spanID;
+    } catch (ex) {
+        console.error('Failed to get parent span ID', ex);
```

Comment:
This error will be printed for the root span of the trace which will throw since filter above returns an empty array.

Reply:
Fixed